RoddyCore/src/de/dkfz/roddy/plugins/LibrariesFactory.groovy

```diff
@@ -238,7 +238,16 @@ public class LibrariesFactory extends Initializable {
      * @param usedPlugins
      */
     public boolean resolveAndLoadPlugins(String[] usedPlugins) {
-        def queue = buildupPluginQueue(loadMapOfAvailablePluginsForInstance(), usedPlugins)
+        def mapOfAvailablePlugins = loadMapOfAvailablePluginsForInstance()
```

Comment:
This code collides with the changes I just yesterday made in https://github.com/eilslabs/Roddy/pull/23